torch/onnx/_internal/exporters/fx_symbolic.py

```diff
@@ -165,59 +165,54 @@ def _module_expansion_symbolic_trace(
             setattr(torch, name, wrapped)
 
 
-@_beartype.beartype
-def export_without_parameters_and_buffers(
-    module: torch.nn.Module,
-    *args,
-    decomposition_table: Optional[Dict[torch._ops.OpOverload, Callable]] = None,
-    use_binary_format: bool = True,
-    opset_version: int = _constants.ONNX_DEFAULT_OPSET,
-    op_level_debug: bool = False,
-    enable_dynamic_axes: bool = True,
-    # kwargs are the keyword arguments to call "module"; that is,
-    # module(*args, **kwargs) must run.
-    **kwargs,
-) -> Tuple[
-    Union[onnx.ModelProto, bytes],
-    torch.fx.GraphModule,
-    Tuple[Any, ...],
-    Tuple[torch.Tensor, ...],
-]:
-    graph_module, bound_args = _trace_into_fx_graph_via_fx_symbolic_trace(
-        module, *args, **kwargs
-    )
+class FXSymbolicTraceExportOutput(ExportOutput):
```

Comment:
Same as above (new stuff not discussed on v1 design proposal which deviates from the complete design)